cluster/juju/layers/kubernetes-worker/templates/default-http-backend.yaml

```diff
@@ -17,7 +17,7 @@ spec:
         # Any image is permissable as long as:
         # 1. It serves a 404 page at /
         # 2. It serves 200 on a /healthz endpoint
-        image: gcr.io/google_containers/defaultbackend:1.0
+        image: {{ backend_image }}
```

Comment:
Shouldn't this be "defaultbackend_image"?

Reply:
Probably.